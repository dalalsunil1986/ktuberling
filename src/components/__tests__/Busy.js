import React from 'react';
import renderer from 'react-test-renderer';

import Busy from '../Busy';


describe('Busy', () => {
  it('render', () => {
    const tree = renderer.create(<Busy />);

    expect(tree).toMatchSnapshot();
  });

  it('render active', () => {
    const tree = renderer.create(<Busy active/>);

    expect(tree).toMatchSnapshot();
  });
});
