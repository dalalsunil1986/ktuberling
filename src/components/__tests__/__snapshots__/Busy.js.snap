// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Busy render 1`] = `null`;

exports[`Busy render active 1`] = `
<div
  className="busy"
>
  <div
    className="alert alert-success"
  >
    Loading ...
  </div>
  <div
    className="overlap"
  />
</div>
`;
